.vsts-ci.yml

```diff
@@ -32,7 +32,8 @@ variables:
     - group: DotNetBuilds storage account read tokens
     - name: _InternalRuntimeDownloadArgs
```

Comment:
The name of this no longer makes sense if you're adding OfficialBuilder to it.  You could either rename it something like _InternalBuildArgs and then update all the places it's used or create a new variable that could be used. @MattGal any opinion here?

I think this is the only leg that actually needs the value to be set: https://github.com/dotnet/sdk/blob/247409ebf601532e528d1baa50b4258bbbfa3454/eng/build.yml#L68 as that's the build we then consume in the installer repo when building the Microsoft official drops.

All the others are used in PR legs and I'm not sure we need telemetry enabled for our own PR builds.

Reply:
Pushed an update based on my above suggestion. Tested with an internal CI build to confirm telemetry was enabled for the official build. The changes LGTM but someone else should take a second look.